app/rss/routes.py

```diff
@@ -0,0 +1,202 @@
+            )
+            fg.link(
+                href=f"{request.url_root}rss/?mt={region}",
+                rel="self"
+            )
+            fg.description(
+                f"{region} - Incidents"
+            )
+        #
+        # This part is needed to be able to
+        # make a request without component_name
+        # and get all incidents for the specified region
+        #
+        incidents = []
+        for component in components:
+            component_data = ComponentSchema().dump(component)
+            incidents.extend(component_data["incidents"])
+
+        for incident in reversed(sorted_incidents(incidents)):
+            if incident["end_date"] is None or datetime.strptime(
+                incident["end_date"], "%Y-%m-%d %H:%M"
+            ) <= datetime.today():
+                fe = fg.add_entry()
+                fe.title(incident["text"])
+                start_date = tz.localize(
+                    datetime.strptime(
+                        incident["start_date"],
+                        "%Y-%m-%d %H:%M"
+                    )
+                )
+                if incident["end_date"]:
+                    end_date = tz.localize(
+                        datetime.strptime(
+                            incident["end_date"],
+                            "%Y-%m-%d %H:%M"
+                        )
+                    )
+                else:
+                    end_date = None
+                content_string = f"Incident impact: {incident['impact']}, \
+                        Incident start date: {start_date}, \
+                        incident end date: {end_date}"
+                #
+                # "updates" exist as a sublist for the incident,
+                # schemes are described in the file:
+                #  "app/api/schemas/components.py"
+                # look at the "class IncidentSchema(Schema):"
+                # and class "IncidentStatusSchema(Schema):"
+                #
+                if incident["updates"]:
+                    for update in incident["updates"]:
+                        update_timestamp = tz.localize(
+                            datetime.strptime(
+                                update["timestamp"],
+                                "%Y-%m-%d %H:%M"
+                            )
+                        )
+                    content_string += f"\n \
+                                        <div class='update'> \
+                                        Update: {update['text']}, \
+                                        Update timestamp: {update_timestamp} \
+                                        </div>"
+                    fe.pubDate(update_timestamp)
+                else:
+                    fe.pubDate(start_date)
+                fe.content(f"<div class='item_desc'>{content_string}</div>")
+
+        rss_string = fg.rss_str(pretty=True).decode("utf-8")
+        rss_string = rss_string.replace("<span", "<div")
```

Comment:
I do not think it is a good idea to do such replacements in the content generated by 3rd party tool outside of control. It should not be necessary also.